conf/messages.cy

```diff
@@ -87,3 +87,7 @@ agentAnswer.heading = Gwirio manylion yr asiantaeth
 trust.nonTaxable.countryOfResidenceYesNo.checkYourAnswersLabel = A ydych yn gwybod beth yw gwlad breswyl {0}?
 trust.nonTaxable.countryOfResidenceInTheUkYesNo.checkYourAnswersLabel =  A yw {0} yn breswyl yn y DU?
 trust.nonTaxable.countryOfResidence.checkYourAnswersLabel = Beth yw gwlad breswyl {0}?
+
+charity.nonTaxable.countryOfResidenceYesNo.checkYourAnswersLabel = A ydych yn gwybod beth yw gwlad breswyl {0}?
+charity.nonTaxable.countryOfResidenceInTheUkYesNo.checkYourAnswersLabel =  A yw {0} yn breswyl yn y DU?
```

Comment:
Just wondering if the double space after the `=` adds a space to the start of the message?

Reply:
Removed